stable/cluster-autoscaler/templates/deployment.yaml

```diff
@@ -183,9 +180,6 @@ spec:
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
       volumes:
-        - name: ssl-certs
-          hostPath:
-            path: {{ .Values.sslCertHostPath }}
         {{- if eq .Values.cloudProvider "gce" }}
```

Comment:
Will you please adjust this, so we don't set `volumes: null` when `ne .Values.cloudProvider "gce"`?